src/scripts/main.js

```diff
@@ -1,7 +1,25 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const notificationTitle = document.createElement('h2');
+
+  notificationTitle.className = 'title';
+  notificationTitle.textContent = title;
+
+  const notificationMessage = document.createElement('p');
+
+  notificationMessage.innerHTML = description;
```

Comment:
Using `innerHTML` can introduce security risks such as XSS (Cross-Site Scripting) if the `description` contains user input. If HTML tags are not needed, consider using `textContent` to avoid these risks.